.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,4 +15,5 @@ Give us some context:
 * It'd also be really handy if you could tell us the contents of your ```version.known``` file
 * What database are you using? (e.g. mongo, mysql, postgres)
 * Any warnings or errors in your admin/diagnostics page?
+* Can you include examples of any Micropub / API calls / webhook pings you make?
```

Comment:
```suggestion
* If this is a programming bug, can you include examples of any Micropub / API calls / webhook pings you make? Otherwise please don't worry about what this means!
```

I think this is too technical for a general bug report. Here's a suggested change of language.

Reply:
Your changes seem sound. Looks like you need to re-approve even though I've accepted your changes....